aarch64_linux/aarch64_ci_setup.sh

```diff
@@ -19,7 +19,16 @@ curl -L -o /mambaforge.sh https://github.com/conda-forge/miniforge/releases/late
 chmod +x /mambaforge.sh
 /mambaforge.sh -b -p /opt/conda
 rm /mambaforge.sh
-/opt/conda/bin/conda config --set ssl_verify False
-/opt/conda/bin/conda install -y -c conda-forge python=${DESIRED_PYTHON} numpy pyyaml setuptools patchelf pygit2 openblas ninja scons
+source /opt/conda/etc/profile.d/conda.sh
```

Comment:
Can you explain why this is necessary/what this changes for the sake of understanding?

Reply:
This is needed to be able to create the env below. Answering there.